lib/datasource/docker/common.ts

```diff
@@ -178,7 +178,10 @@ export function getRegistryRepository(
   }
   let repository = split.join('/');
   if (!registry) {
-    registry = registryUrl;
+    registry = registryUrl.replace(
+      'https://docker.io',
+      'https://index.docker.io'
+    );
```

Comment:
Isn't it easier to combine this with next if condition? 🤔

Reply:
No, because the next condition is for when it's embedded in the `lookupName`. I added this `if` kind of independently of this feature just because I made the mistake of settings `registryUrls: ['https://docker.io']` and I figured others might too.